lib/nodeping/client.go

```diff
@@ -235,6 +212,37 @@ func (c *Client) GetUptimesForChecks(checkIDs map[string]string, period Period)
 	return uptimes
 }
 
+func (c *Client) request(path string, v any) error {
```

Comment:
Since this is only doing GET, it might be better to call it that.

```suggestion
func (c *Client) get(path string, v any) error {
```